### PR TITLE
Adjust Python Package Metadata query

### PR DESCRIPTION
**Pull request by pacospace**
- Added missing metadata to the output of get Python Package metadata query

Fixes: https://github.com/thoth-station/user-api/issues/680

```
Test

FUNCTION: get_python_package_version_metadata

INPUTS: {'index_url': 'https://tensorflow.pypi.thoth-station.ninja/index/manylinux2010/AVX2/simple', 'package_version': '2.0.0', 'package_name': 'tensorflow'}

RESULTS: {
'author': 'Red Hat Inc.', 
'author_email': 'smodeel@redhat.com', 
'download_url': 'https://github.com/tensorflow/tensorflow/tags', 
'home_page': 'https://www.tensorflow.org/', 
'keywords': 'tensorflow tensor machine learning', 
'license': 'Apache 2.0', 
'maintainer': None, 
'maintainer_email': None, 
'metadata_version': '2.1', 
'name': 'tensorflow', 
'summary': 'TensorFlow is an open source machine learning framework for everyone.', 
'version': '2.0.0', 
'requires_python': None, 
'description': None, 
'description_content_type': None, 
'classifiers': ['Intended Audience :: Developers', 'Topic :: Software Development :: Libraries :: Python Modules', 'Programming Language :: Python :: 2.7', 'Programming Language :: Python :: 2', 'Topic :: Software Development', 'Programming Language :: Python :: 3', 'Programming Language :: Python :: 3.4', 'Development Status :: 5 - Production/Stable', 'License :: OSI Approved :: Apache Software License', 'Programming Language :: Python :: 3.5', 'Programming Language :: Python :: 3.6', 'Topic :: Software Development :: Libraries', 'Programming Language :: Python :: 3.7', 'Intended Audience :: Science/Research', 'Intended Audience :: Education', 'Topic :: Scientific/Engineering', 'Topic :: Scientific/Engineering :: Mathematics', 'Topic :: Scientific/Engineering :: Artificial Intelligence'],
'platform': ['CentOS release 6.10 (Final)'],
'supported_platform': [], 
'requires_external': [], 
'project_url': [], 
'provides_extra': [], 
'requires_dist': ['absl-py (>=0.7.0)', 'astor (>=0.6.0)', 'gast (==0.2.2)', 'google-pasta (>=0.1.6)', 'keras-applications (>=1.0.8)', 'keras-preprocessing (>=1.0.5)', 'numpy (<2.0,>=1.16.0)', 'opt-einsum (>=2.3.2)', 'six (>=1.10.0)', 'protobuf (>=3.6.1)', 'tensorboard (<2.1.0,>=2.0.0)', 'tensorflow-estimator (<2.1.0,>=2.0.0)', 'termcolor (>=1.1.0)', 'wrapt (>=1.11.1)', 'grpcio (>=1.8.6)', 'wheel (>=0.26)', 'backports.weakref (>=1.0rc1); python_version < "3.4"', 'enum34 (>=1.1.6); python_version < "3.4"'], 
'provides_dist': [], 
'obsolete_dist': []}
0.1498656830008258
```

Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

**Comment by pacospace** [comment]
Updated :)

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -3375,6 +3392,50 @@ def get_python_package_versions_count_all(
 
             return query.count()
 
+    def get_multi_values_key_python_package_version_metadata(
```
Comment (fridex) [inline thoth/storages/graph/postgres.py]: Can this method be private?

Comment (fridex) [reply thoth/storages/graph/postgres.py]: And also `MULTI_VALUE_KEY_PYTHON_PACKAGE_METADATA_MAP`.

Comment (pacospace) [reply thoth/storages/graph/postgres.py]: Updated, thanks :)

**Pull request merged**